src/index.ts

```diff
@@ -1,7 +1,7 @@
 import Web3 from 'web3';
 import { StorageProvider, IpfsStorageProvider } from './storage';
 import ContractsManager from './contracts';
-import { ProofPointsRepo } from './proofPoints';
+import { ProofPointsRepo, ProofPointIssueResult, ProofPointValidateResult, ProofPointValidity } from './proofPoints';
```

Comment:
Just curious, is there any way to scope all these things under a `ProofPoint` module or namespace? And then just return that, rather than listing all these classes?

Reply:
yes thats what the export at the end of this file does

Reply:
I mean so they are defined as `ProofPoint::Status`, `ProofPoint::IssueResult`, etc...